src/applications/disability-benefits/2346/helpers/index.js

```diff
@@ -71,3 +72,20 @@ export function validateZIP(errors, zip) {
     );
   }
 }
+
+export const newAddressHider = formData => {
+  const address = get('newAddress', formData) || {};
+  // return true if living at view:militaryBaseInfo object
+  return Object.keys(address).every(entry => {
+    const value = address[entry] || '';
+    return typeof value === 'string' ? value === '' : true;
+  });
+};
+
+export const updateRadioLabels = (formData, name) => {
```

Comment:
May want to explore omitting/giving love to this function, too 🙂

Reply:
💯 function love incoming